tarantool.moon

```diff
@@ -10,21 +10,21 @@ string = string
 socket = nil
 decode_base64 = nil
 sha1_bin = nil
-crypto = nil
+sha1 = nil
 
 -- Use non NGINX modules
 -- requires: luasock (implicit), lua-resty-socket, sha1
 
 openssl_sha1_hash = (msg) ->
-  crypto.digest('sha1', msg, true)
+  sha1.binary(msg)
 
 if not ngx then
   socket = require("socket")
   socket.unix = require("socket.unix")
   mime   = require("mime")
   decode_base64 = mime.unb64
-  crypto = require("crypto")
-  if crypto.sha1
+  sha1 = require("sha1")
+  if not sha1.binary
     print "This version of SHA1 is text only and is not supported"
   else
     sha1_bin = openssl_sha1_hash
```

Comment:
AFAIS `sha1.binary` is in the https://github.com/mpeterv/sha1 moduel from the initial commit, so I don't see a point to check it for existence.

Reply:
I also thought about it, but decided to leave it for correctness (with the old logic).

Reply:
It would make sense if we would fallback to `require('crypto')`, but we don't. As for me, the fallback is not necessary. I don't even think that we really bother about performance here, because we need to calculate a hash function once for a connection: during authentication.

Reply:
Okay, then I'll remove this check.